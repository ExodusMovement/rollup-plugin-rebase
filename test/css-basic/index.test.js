import { bundle, clean, list, read } from "../util"

const root = __dirname

test("Flat Single", async () => {
  await bundle(root, "index.js", "output/index.js")

  expect(list(root, "output")).resolves.toMatchSnapshot()
  expect(read(root, "output/index.js")).resolves.toMatchSnapshot()
  expect(read(root, "output/KKaZlCcl.css")).resolves.toMatchSnapshot()

  await clean(root, "output")
})
