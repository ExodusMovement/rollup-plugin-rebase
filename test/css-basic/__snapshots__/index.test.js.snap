// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flat Single 1`] = `
"KKaZlCcl.css
KKaZlCcl.css.map
bsdudxaF.md
index.js"
`;

exports[`Flat Single 2`] = `
"import content from './bsdudxaF.md';
import css from './KKaZlCcl.css';

function index() {
  return content + css
}

export default index;
"
`;

exports[`Flat Single 3`] = `
"body {
  font-size: 20px;
}

/*# sourceMappingURL=KKaZlCcl.css.map */"
`;
